.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,175 @@
+# This workflow is the main workflow for regenerating the benchmarks data needed for Bowtie's UI.
+# It runs all benchmarks over Bowtie's supported implementations, publishing the benchmark reports for use in the frontend.
+name: Collect New Benchmark Results
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Every Monday at 00:00 UTC
+    - cron: "0 0 * * 1"
```

Comment:
Try to avoid running anything at midnight that really doesn't need to run then. Think about all the people on the planet who've chosen that time of day to run stuff thinking it's "arbitrary". So even though GitHub may try to prevent degradation at that time, perhaps knowing how many people have chosen it, there's still no reason to take the risk.

Reply:
Configured it to run at 8 in the morning